src/plugins/intel_gpu/src/graph/crop.cpp

```diff
@@ -254,6 +254,11 @@ void crop_inst::on_execute() {
 }
 
 void crop_inst::reuse_input() {
+    update_output_memory();
+}
+
+void crop_inst::update_output_memory() {
```

Comment:
The behavior is slightly different with other `update_output_memory`. What about keeping them same?

Reply:
Do you mean that it's better to rollback reuse_input() function and add memory address checking to update_output_memory()?